examples/heat-equation/doc/intro.dox

```diff
@@ -0,0 +1,39 @@
+<a name="Intro"></a>
+<h1>Introduction</h1>
+This example solves a 2D heat conduction equation
+
+$
+    u : [0, d]^2 \rightarrow R\\
+    \partial_t u = \delta u + f
+$
+
+with Dirichlet boundary conditions and given initial condition and function f.
+
+The partial differential equation (PDE) is solved with a finite difference
+spatial discretization on an equidistant grid: For `n` grid points,
+and grid distance $h = 1/n$ we write
+
+$
+    u_{i,j}' =
+    \alpha\frac{u_{i-1,j}+u_{i+1,j}+u_{i,j-1}+u_{i,j+1}-4u_{i,j}}{h^2}+f_{i,j}
+$
+
+We then build an implicit Euler integrator by discretizing with time step $\tau$
+
+$
+    \frac{u_{i,j}^{k+1} - u_{i,j}^k}{\tau} =
+    \alpha\frac{u_{i-1,j}^{k+1}+u_{i+1,j}^{k+1}
+               -u_{i,j-1}^{k+1}-u_{i,j+1}^{k+1}+4u_{i,j}^{k+1}}{h^2}
+    +f_{i,j}^{k+1}
+$
```

Comment:
Is it worth adding somewhere that this example uses a constant-in-time source?  Denoting f_{i,j}^{k+1} isn't incorrect, of course, but it perhaps implies that f will be changing with time, which it is not.

Reply:
That's if course correct, thanks for pointing it out!